smoke-test/src/test/java/org/opennms/smoketest/minion/DetectorsCommandTest.java

```diff
@@ -0,0 +1,158 @@
+            + "MSExchange: org.opennms.netmgt.provision.detector.msexchange.MSExchangeDetector\n"
+            + "MX4J: org.opennms.netmgt.provision.detector.jmx.MX4JDetector\n"
+            + "Memcached: org.opennms.netmgt.provision.detector.simple.MemcachedDetector\n"
+            + "NOTES: org.opennms.netmgt.provision.detector.simple.NotesHttpDetector\n"
+            + "NRPE: org.opennms.netmgt.provision.detector.simple.NrpeDetector\n"
+            + "NSClient: org.opennms.protocols.nsclient.detector.NsclientDetector\n"
+            + "NTP: org.opennms.netmgt.provision.detector.datagram.NtpDetector\n"
+            + "OMSAStorage: org.opennms.netmgt.provision.detector.snmp.OmsaStorageDetector\n"
+            + "PERC: org.opennms.netmgt.provision.detector.snmp.PercDetector\n"
+            + "POP3: org.opennms.netmgt.provision.detector.simple.Pop3Detector\n"
+            + "RadiusAuth: org.opennms.protocols.radius.detector.RadiusAuthDetector\n"
+            + "SMB: org.opennms.netmgt.provision.detector.smb.SmbDetector\n"
+            + "SMTP: org.opennms.netmgt.provision.detector.simple.SmtpDetector\n"
+            + "SNMP: org.opennms.netmgt.provision.detector.snmp.SnmpDetector\n"
+            + "SSH: org.opennms.netmgt.provision.detector.ssh.SshDetector\n"
+            + "TCP: org.opennms.netmgt.provision.detector.simple.TcpDetector\n"
+            + "TrivialTime: org.opennms.netmgt.provision.detector.simple.TrivialTimeDetector\n"
+            + "WEB: org.opennms.netmgt.provision.detector.web.WebDetector\n"
+            + "WMI: org.opennms.netmgt.provision.detector.wmi.WmiDetector\n"
+            + "WS-Man: org.opennms.netmgt.provision.detector.wsman.WsManDetector\n"
+            + "Win32Service: org.opennms.netmgt.provision.detector.snmp.Win32ServiceDetector\n"
+            + "XMP: org.opennms.netmgt.protocols.xmp.detector.XmpDetector";
+
+    @ClassRule
+    public static final TestEnvironment getTestEnvironment() {
+        if (!OpenNMSSeleniumTestCase.isDockerEnabled()) {
+            return new NullTestEnvironment();
+        }
+        try {
+            final TestEnvironmentBuilder builder = TestEnvironment.builder().all();
+            OpenNMSSeleniumTestCase.configureTestEnvironment(builder);
+            m_testEnvironment = builder.build();
+            return m_testEnvironment;
+        } catch (final Throwable t) {
+            throw new RuntimeException(t);
+        }
+    }
+
+    @Before
+    public void checkForDocker() {
+        Assume.assumeTrue(OpenNMSSeleniumTestCase.isDockerEnabled());
+    }
+
+    @Test
+    public void LoadDetectorsOnMinionTest() throws Exception {
+
+        final InetSocketAddress sshAddr = m_testEnvironment.getServiceAddress(ContainerAlias.MINION, 8201);
+        try (final SshClient sshClient = new SshClient(sshAddr, "admin", "admin");) {
+            PrintStream pipe = sshClient.openShell();
+            pipe.println("provision:list-detectors");
+            Thread.sleep(2000);
```

Comment:
We should grab the output using the same pattern used here: https://github.com/OpenNMS/opennms/blob/54c153ed33867efcf8bb9eacbad75345246f02f7/smoke-test/src/test/java/org/opennms/smoketest/minion/SyslogTest.java#L122

This avoids requiring the explicit sleep.